libs/template/templates/lakeflow-pipelines/databricks_template_schema.json

```diff
@@ -1,57 +1,96 @@
 {
-    "welcome_message": "\nWelcome to the template for Lakeflow Declarative Pipelines!",
+    "//": "This template is based on an upcoming version of the default-python template, but comes with a SQL variation + certain defaults that are specific to Lakeflow Pipelines.",
```

Comment:
Note to self: We have to test the new template schema with the workspace creation UI to see if everything still makes sense and looks OK.

cc @ilyakuz-db